package-lock.json

```diff
@@ -0,0 +1,14380 @@
+{
```

Comment:
Do not add "package-lock.json" to the project. It was removed intentionally. We use yarn.lock.